recipes/aicspylibczi/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "aicspylibczi" %}
+{% set version = "3.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aicspylibczi-{{ version }}.tar.gz
+    sha256: b8c2e9360e47ad9c08d9af492bea532daa93f6b73d596fadf683230b212c24e7
+  - url: https://github.com/zeiss-microscopy/libCZI/archive/0eb12ef9612ec31f6715fdf631f5c2f86a54115a.zip
+    sha256: 95f4da7a22bf026d0f31cac1b67a7678dd9f72192e100c21600cebb45514a456
+    folder: libCZI
+  - url: https://github.com/pybind/pybind11/archive/45f792efdd92da094548e2095d6efdbfa7e536ee.zip
+    sha256: b9eb37eb1dd9161e387a2b10fd1ed70e3ad0c3e1c2cb3e75a24f4c41b8fe45ca
+    folder: pybind11
```

Comment:
suggest removing the last two source entries as their code should now be included in the sdist of the main package